website/docs/intro/install/docker-pull.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Copyright (c) The OpenTofu Authors
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (c) 2023 HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+
```

Comment:
Since these go into an example, the comments make the snippet rather lengthy. I would suggest removing both the hashbang line and the copyright headers.

![image](https://github.com/opentofu/opentofu/assets/86970079/076375c5-70b7-4bb0-95c8-32d944f21d08)


```suggestion
```